members/views.py

```diff
@@ -18,7 +18,8 @@
 from django.views import View
 
 from core.utils import validate_captcha
-from members.forms import SignUpForm, AlumniSignUpForm, FunctionaryForm
+from members.forms import SignUpForm, AlumniSignUpForm, FunctionaryForm, MemberEditForm
+from . import forms
```

Comment:
Seems like an  unused import, also could be confused with how we usually import the django forms module